src/main/scala/github/sync/Main.scala

```diff
@@ -19,16 +21,22 @@ object Main extends IOApp.WithContext {
   override protected def executionContextResource: Resource[SyncIO, ExecutionContext] = Resource.liftF(SyncIO(ec))
 
   private def main: Opts[IO[ExitCode]] =
-    Config.opts.map { case Config(token, source, target, deleteAdditional, dryRun, isVerbose) =>
+    Config.opts.map { case Config(token, source, targets, deleteAdditional, dryRun, isVerbose) =>
       BlazeClientBuilder[IO](ec).resource.use { client =>
         val configuredClient = if (isVerbose) Logger(logHeaders = false, logBody = false)(client) else client
         val github           = new GithubClient[IO](configuredClient, token)
         val program          = new SyncProgram[IO](github)
 
-        program
-          .syncLabels(source, target, deleteAdditional, dryRun)
-          .as(ExitCode.Success)
-          .handleErrorWith(e => Printer[IO].putStrLn(s"Something went wrong: ${e.getMessage}") as ExitCode.Error)
+        targets
+          .traverse(program.syncLabels(source, _, deleteAdditional, dryRun).attempt.map(_.toValidatedNec))
```

Comment:
Attempt each sync and collect all errors at the end, so one bad repo doesn't block the others